pages/about.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import About from './about.page';
+
+describe('<About />', () => {
+  it('should render without crashing', async () => {
+    // Act
+    render(<About />);
+
+    // Assert
+    expect(
+      screen.getByRole('heading', { name: 'about:about' })
```

Comment:
```suggestion
      screen.queryByRole('heading', { name: 'about:about' })
```

is better for expect assertions, since it'll return `null`, rather than throw. `getBy*` is better for fetching elements in the middle of a test for other purposes (triggering a button press etc), because the throwing will stop the test if the premise (can't even get the button) is flawed

Reply:
Thanks for the explanation. Makes sense :)